src/runtime/node/console/index.ts

```diff
@@ -0,0 +1,60 @@
+import type console from "node:console";
+import mock from "../../mock/proxy";
+
+const _console = globalThis.console;
+
+const noop = () => {};
+
+const log: typeof console.log = _console?.log ?? noop;
+const info: typeof console.info = _console?.info ?? log;
+const warn: typeof console.warn = _console?.warn ?? log;
+const debug: typeof console.debug = _console?.debug ?? log;
+const error: typeof console.error = _console?.error ?? log;
+
+const assert: typeof console.assert = _console?.assert ?? noop;
+const clear: typeof console.clear = _console?.clear ?? noop;
+const count: typeof console.count = _console?.count ?? noop;
+const countReset: typeof console.countReset = _console?.countReset ?? noop;
+const dir: typeof console.dir = _console?.dir ?? noop;
+const dirxml: typeof console.dirxml = _console?.dirxml ?? noop;
+const group: typeof console.group = _console?.group ?? noop;
+const groupEnd: typeof console.groupEnd = _console?.groupEnd ?? noop;
+const groupCollapsed: typeof console.groupCollapsed =
+  _console?.groupCollapsed ?? noop;
+const profile: typeof console.profile = _console?.profile ?? noop;
+const profileEnd: typeof console.profileEnd = _console?.profileEnd ?? noop;
+const table: typeof console.table = _console?.table ?? noop;
```

Comment:
this could be safely fallback to `log` table would be array of arrays instead of pretty output